girder_annotation/girder_large_image_annotation/models/annotation.py

```diff
@@ -450,6 +450,46 @@ class AnnotationSchema:
         ]
     }
 
+    overlaySchema = {
+        '$schema': 'http://json-schema.org/schema#',
+        'type': 'object',
+        'properties': {
+            'girderId': {
+                'type': 'string',
+                'description': 'Girder item ID containing the image to '
```

Comment:
We should enforce that this is a 24 digit hexadecimal number `'pattern': '^[0-9a-f]{24}$'`.

Reply:
Fixed in 95c8b45